### PR TITLE
Feature/enable rdf

### PR DESCRIPTION
**Pull request by straleyb**
Adds RDF::URI Support but does not grab labels yet.

### CODE REVIEW

File: app/views/batch/_metadata.html.erb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<h2>Individual Titles  </h2>
+<div id="" class="well">
+  <div class="row">
+    <div class="col-sm-8">
+      <h3>Applies to individual files uploaded</h3>
+      <%= content_tag :p, t('sufia.batch.help.title'), class: "help-block" %>
+      <% @batch.generic_files { |a,b| a.label.downcase <=> b.label.downcase }.each_with_index  do |gen_f, index| %>
```
Comment (straleyb) [inline app/views/batch/_metadata.html.erb]: This is where that pesky line of code that calls #sort would be

@batch.generic_files.sort was the original code. Would overriding this file be a good way to fix this? Or should we send another PR  to Sufia to fix that. 

It seems like ActiveFedora does not want #sort being used on relations. 

https://github.com/projecthydra/active_fedora/blob/master/lib/active_fedora/relation/delegation.rb#L10-L14

Comment (terrellt) [inline app/views/batch/_metadata.html.erb]: #sort should work, just not #sort!

Comment (straleyb) [inline app/views/batch/_metadata.html.erb]: https://github.com/projecthydra/sufia/blob/v6.0.0/app/views/batch/_metadata.html.erb#L7

The version of sufia we are using uses #sort!. Time to upgrade sufia?

Comment (terrellt) [inline app/views/batch/_metadata.html.erb]: :+1:

**Pull request merged**